docs/source/navroot.md

```diff
@@ -0,0 +1,61 @@
+---
+html_meta:
+  "description": ""
+  "property=og:title": "Navigation Root"
+  "keywords": "Plone, plone.restapi, REST, API, site, navigation root"
+---
+
+
+## Navigation root
+
+Plone has an idea called `Navigation Root` which provides a way to root catalog queries, searches, breadcrumbs, etc. in a given section of the site. This feature is useful when working with subsites or multilingual sites, because allows the site manager to somehow restrict where the searches or navigation queries start on the site.
```

Comment:
Please create a term in glossary.md, and we can merge it into the main documentation as well.

```suggestion
Plone has an idea called {term}`navigation root` which provides a way to root catalog queries, searches, breadcrumbs, etc. in a given section of the site.
This feature is useful when working with subsites or multilingual sites, because it allows the site manager to restrict searches or navigation queries to a specific location in the site.
```